app/models/movie.rb

```diff
@@ -3,23 +3,25 @@
 # Table name: movies
 #
 #  id         :bigint           not null, primary key
-#  title      :text
 #  contents   :text
 #  desc       :text
+#  genre      :string
+#  position   :integer
+#  title      :text
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
-#  genre      :string
 #
 
 class Movie < ApplicationRecord
+  acts_as_list
   validates :title, presence: true
   validates :contents, presence: true
   validates :genre, presence: true
 
   # 1ページの動画表示件数を指定
   PER_PAGE = 10
   # ナビゲーションバーで特に指定しない動画のジャンルをまとめて格納
-  PROGRAMMING = ["Basic", "Git", "Ruby", "Ruby on Rails"].freeze
+  GENRE = ["Basic", "Git", "Ruby", "Ruby on Rails", "Money", "Design", "Live", "Talk", "Php", "Writing", "Marketing", "Movie"].freeze
```

Comment:
ここは戻しておいた方がよいと思うのですがどうでしょう？
```suggestion
  PROGRAMMING = Settings.programming.rails.split(", ").freeze
```